source/scripts/Scene.js

```diff
@@ -10,6 +11,7 @@ export default class Scene extends Pixi.Container {
         super()
 
         this.addChild(this.player = new Player())
+        this.addChild(this.foe = new Foe())
```

Comment:
GHYE GHE GHE HE HE HE. Sure, bro. I was gonna call them "Enemy" but I didn't like the sound of that. I'm glad you also have a preferred name to Enemy XD